WalletWasabi.Fluent/ViewModels/Settings/GeneralSettingsTabViewModel.cs

```diff
@@ -71,6 +84,24 @@ public GeneralSettingsTabViewModel()
 		public IEnumerable<FeeDisplayFormat> FeeDisplayFormats =>
 			Enum.GetValues(typeof(FeeDisplayFormat)).Cast<FeeDisplayFormat>();
 
+		private void ModifyRegistry(bool changedOption)
+		{
+			string keyName = "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run";
+			using RegistryKey key = Registry.CurrentUser.OpenSubKey(keyName, true);
+			if (changedOption)
+			{
+				string pathToExe = Assembly.GetExecutingAssembly().Location;
+				pathToExe = pathToExe.Remove(pathToExe.Length - 11);        // This part has to change if this gets released
+				pathToExe += ".Fluent.Desktop.exe";
+
+				key.SetValue("WasabiWallet", pathToExe);
+			}
+			else
+			{
+				key.DeleteValue("WasabiWallet");
+			}
+		}
+
```

Comment:
This should not be in this ViewModel, because this code is not related to the UI.
Would be nice if you could move this into a helper class and would be nicer if you do this in `WasabiWallet` project.

Also make sure this can be only executed on Windows platform.

I can imagine something like this:
```
namespace WalletWasabi.Helpers
{
    public static class StartupHelpers
	{
	    public static void ModifyStartupSetting(bool value)
	    {
            if (Windows)
            {
                ModifyRegistry(bool value)
            }
            if (Linux)
            {
                // Method call here
            }
            if (Macos)
            {
                // Method call here
            }
        }
	}
	
	private void ModifyRegistry(bool changedOption)
    {
        .....
    }
}
```

Reply:
Make sense, i will replace it.